docs: update readme (#121)
Open-Attestation/decentralized-renderer-react-components: README.md

### FrameConnector

This component will create a frame and establish a connection with the provided decentralized renderer. Its properties include:

- `source`: The URL to the decentralized renderer that will handle the document to display
- `dispatch`: The function listening for actions triggered by the decentralized renderer
- `onConnected`: The function being called when the connection to the decentralized renderer has been established. The function will be provided as the first parameter toe another, which can be used for sending actions to the iframe.

Please check the code in `example/application` to see how to use this component. You can also start the example application using the command `npm run example:application`

### FramedDocumentRenderer

This component will establish a connection with a host embedding the application within an iframe. Its properties include:

- `templateRegistry`: The configuration of the templates handled by the decentralized renderer. `templateRegistry` is an object where each key holds an array of `Template Configuration`. 

  One `Template Configuration` consists of:

>   - `id`: A unique identifier withing the current array of the template

Author: sorry for typo - fixed

>   - `label`: A string to represent what the template is

Author: added

- It will automatically provide the available templates when a document has been requested for rendering (i.e. it will call the `UPDATE_TEMPLATES` action once the document has been rendered).
- it will automatically call the `OBFUSCATE` action when it is requested by a `Template`.

You need to check the code in `example/decentralized-renderer` to see how to use this component. You can also start the example application using the command `npm run example:renderer`

#### Template (React component)

When a document is being rendered, each configured `Template` will be provided with the following properties:

- `document`: (mandatory) Document data as returned by the `getData` method from [@govtechsg/open-attestation](https://github.com/Open-Attestation/open-attestation)
- `rawDocument`: (optional) OpenAttestation document
- `handleObfuscation`: (mandatory) A function to call that will handle obfuscation in the document. The value provided must follow the path property as handled by [lodash#get](https://lodash.com/docs/4.17.15#get).

## Development

Run the following commands for different development tasks:

> - `npm run storybook`: to start a storybook, create stories, and visualize the different component

Reviewer: ```suggestion
- `npm run storybook`: to start a storybook, create stories, and visualize different components
```

Author: updated

- `npm run test`: to run tests
- `npm run lint`: to run lint

> - `npm run example:application`: to run the example build with the library to develop an hosting application. 

Reviewer: ```suggestion
- `npm run example:application`: to run an example application built with this library. 
```

Author: updated

  Be sure to update the example if you update this library.

> - `npm run example:renderer`: to run the example build with the library to develop a decentralized renderer. 

Reviewer: ```suggestion
- `npm run example:renderer`: to run an example decentralized renderer built with this library. 
```

Author: updated
